bee-node/build_and_run.sh

```diff
@@ -0,0 +1,78 @@
+# docker image tag from Cargo.toml in this way the version will be in sync
```

Comment:
maybe there should be a `#!/bin/sh` here? I cannot run the script in linux as described in usage without setting the shell and doing `chmod u+x build_and_run.sh`

Reply:
I added `#!/bin/sh` as you suggested and corrected `build_and_run.sh run bee-image -> create bee node docker image` to `build_and_run.sh run bee-image -> run a bee node docker container instance`

Reply:
yes you have to give to your user execution privileges over the script. On `linux/macos` you can do that with `chmod u+x <filename>`.